Tests/FluentAssertions.Specs/Specialized/TaskOfTAssertionSpecs.cs

```diff
@@ -308,8 +308,8 @@ await act.Should().ThrowAsync<ArgumentOutOfRangeException>()
                 .WithMessage("* value of pollInterval must be non-negative*");
         }
 
-        [Fact]
-        public async Task When_no_exception_should_be_thrown_async_for_null_after_wait_time_it_should_throw()
+        [Fact] // TODO What is the relevant difference to When_subject_is_null_it_should_fail?
```

Comment:
I added both methods in #1289, but with today's eyes I cannot see what purpose _this_ method serves.
`When_subject_is_null_it_should_fail` also verifies that `because` uses the `objectArgs`.

Reply:
removed